pkgs/by-name/ir/ironbar/package.nix

```diff
@@ -56,16 +64,31 @@ rustPlatform.buildRustPackage rec {
     gtk3
   ];
 
-  preFixup = ''
-    gappsWrapperArgs+=(
+  runtimeDeps = [
+    luajitPackages.lgi
+  ];
+
+  buildNoDefaultFeatures = if features == [] then false else true;
```

Comment:
I had this same mistake, but this expression can be simplified to just be `features != []`

Reply:
Ahh, good catch.

I've amended this in https://github.com/NixOS/nixpkgs/pull/307726/commits/07e1d0b4aa1384a7020e6f613c43b76b2965ff12